### ISSUE TITLE
There are three people in the recipient list, but there are only two people on the transfer selection recipient page

### ISSUE THREAD

**Issue by wanghuijiao** [AyoEasy/Easylink-Mobile-IOS#89]
https://github.com/AyoEasy/Easylink-Mobile-IOS/assets/152844804/1cbd0d89-2faa-406f-9ee6-1a016cd7d98b

**Comment by MN3280** [comment]
@wanghuijiao Are those 3 recipients came from the same country and currency? Because we filter user on transfer list based on country and currency, if the country OR currency are different, it will show different list

**Comment by wanghuijiao** [comment]
> Are those 3 recipients came from the same country and currency? Because we filter user on transfer list based on country and currency, if the country OR currency are different, it will show different list

<img width="339" alt="image" src="https://github.com/AyoEasy/Easylink-Mobile-IOS/assets/152844804/67fb194e-eda1-40f6-a236-68b723730c71">
All three are from China

**Comment by MN3280** [comment]
china got 2 currency , USD and CNY, Are those 3 recipients came from the same currency?

**Comment by wanghuijiao** [comment]
> china got 2 currency , USD and CNY, Are those 3 recipients came from the same currency?

Yes, you can log in to my account and try it out. Account: 855555555, password: 123123Whj?

**Comment by wanghuijiao** [comment]
> after I check, you got 2 recipient from ewallet and 1 from bank , that's why it only show 2 data when its on Alipay(e-wallet) and the other one is at the bank acc. If you don't understand you can confirm it to hammer .

Okay, I understand
